api/envoy/service/tap/v2alpha/common.proto

```diff
@@ -64,9 +67,17 @@ message MatchPredicate {
 
     // HTTP response trailers match configuration.
     HttpHeadersMatch http_response_trailers_match = 8;
+
+    ClusterMatch destination_cluster_match = 9;
```

Comment:
How is this populated/matched?

Reply:
right now it checks that the destination cluster name is one of those defined in `ClusterMatch`.
I can change `ClusterMatch` to have a StringMatcher for more flexibility in matching.

My thinking here was to allow limiting taps to specific services.